cmake/config-build.cmake

```diff
@@ -44,7 +44,9 @@ if(RTS_BUILD_GENERALS)
     add_feature_info(GeneralsDocs RTS_BUILD_GENERALS_DOCS "Build Generals Documentation")
 endif()
 
-if(NOT IS_VS6_BUILD)
+if(IS_VC6_BUILD)
+    target_compile_definitions(core_config INTERFACE defined(VC6_BUILD))
```

Comment:
This line should be `target_compile_definitions(core_config INTERFACE VC6_BUILD)`

Reply:
This happened on mass rename :D